candi/cloud-providers/aws/bashible/common-steps/all/000_install_nvme_cli.sh.tpl

```diff
@@ -12,4 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-bb-yum-install nvme-cli
+case $(bb-is-bundle) in
+  debian|ubuntu-lts) bb-apt-install nvme-cli ;;
+  centos) bb-yum-install nvme-cli ;;
+esac
```

Comment:
Can we self-compile static nvme-cli ?

Reply:
nope, static build didn't work for this package.